_best_practices/06-code_conduct.md

```diff
@@ -1,29 +1,45 @@
 category: Governance
-index: 40
+index: 39
 difficulty: na
 references: [TTAID, AIHLEG] #
 comments: True
 description:
 image: #
 photocredit: #
 
-intent: Build trust and ensure adherence to ethical principles and values. #
-motivation: ML and AI can severly impact human lives, usually without  malicious intentions.   #
-applicability: Code of conducts should be applied to any machine learning application. #
-related: [tradeoff,audits, interpretable] #
+#intent: Build trust and ensure adherence to ethical principles and values. #
+#intent: Make explicit to all stakeholders what the objectives and constraints of your ML application are #
+intent: Explicitly align all stakeholders on the ethical values and constraints of your ML application
+#motivation: ML and AI can severly impact human lives, usually without  malicious intentions.   #
+#motivation: By setting explicit goverance objectives, all stakeholders can align on the constraints and goals that the team and their product can be held accountable for. #
+motivation: ML applications can severly impact human lives. Avoiding negative impacts, even without malicious intent, requires all stakeholders operate against the same ethical values.
+#applicability: Code of conducts should be applied to any machine learning application. #
+#applicability: Governance objectives should be set for any production-level ML application.
+applicability: Values for Responsible AI should be established for any ML application.
+related: [tradeoff,audits,interpretable] #
 dependencies: #
 survey_question: Q99 #
 survey_item: Our organisation subscribes to a code of conduct for Responsible AI.
 ---
 
-A code of conduct helps to share ethical principles and values across organisations.
-Moreover, defining or subscribing to a code of conduct helps to build trust with users and enhances the auditability of the application.
+A good starting point for sharing ethical values accross organisations is to subscribe to a code of conduct. You can create one specific for your situation, or you can refer to a general governance framework.
 
-A code of conduct can be regarded as a chart which guides all applications developed inside an organisation.
-Responsible AI can guard against the use of biased algorithms (or data) and ensure that decisions made by ML models can be justified and explainable.
+General governance frameworks that you may want to refer to include:
+- <a href="https://ai.google/responsibilities/responsible-ai-practices">Google Responsible AI</a>
+- <a href="https://www.microsoft.com/en-us/ai/responsible-ai">Microsoft AI principles</a>
+- <a href="https://ec.europa.eu/digital-single-market/en/news/ethics-guidelines-trustworthy-ai">European Commission High-Level Expert Group - Ethical guidelines for trustworthy AI</a>
+
+These frameworks set high-level objectives for concerns such as security, privacy, and fairness. They can be complemented or refined for your specific situation.
```

Comment:
Shall we also cite the EU Trustworthy AI guidelines? I think that is more comprehensive on the different levels of AI, i.e. lawful, ethical, robust